modules/core/test/test_mat.cpp

```diff
@@ -9,6 +9,10 @@
 #include "opencv2/core/eigen.hpp"
 #endif
 
+#if defined(HAVE_CUDA)
+    #include <cuda_runtime.h>
+#endif
```

Comment:
It make sense to create new file `test_cuda.cpp` for this test

Reply:
ok. I will move it into a new file.